test/e2e/set/scyllacluster/scyllacluster_scaling.go

```diff
@@ -211,4 +216,65 @@ var _ = g.Describe("ScyllaCluster", func() {
 		verifyCQLData(ctx, diRF2)
 		verifyCQLData(ctx, diRF3)
 	})
+
+	g.It("should support vertical scaling", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
+
+		const membersCount = 3
```

Comment:
is there any reason why this resource consuming test is using 3 nodes instead just 1?